src/pages/es/core-concepts/astro-components.md

```diff
@@ -41,7 +41,7 @@ import Button from './Button.astro';
 ```
 
 
-### Script del componente
+### Script de un componente
```

Comment:
Looks like a few other pages were linking to this section using the `#script-del-componente` fragment. Could you update those pages to use `#script-de-un-componente`?

Reply:
Will do, thanks.